client/src/components/scores/score.js

```diff
@@ -3,51 +3,57 @@
 export default {
 	template: `
 	<div class="card-section" ng-class="{ loading: loading }">
-	<h4>
-		<a editable-select="score.data.teamNumber" buttons="no" blur="submit" onaftersave="score.save()"

```

Comment:
I don't know if you've noticed but all spaces in the program has changed... Now this isn't a big deal, just something to notice and look into why it's happenning